src/main/resources/testnet.conf

```diff
@@ -8,10 +8,15 @@ ergo {
     # than headerChainDiff blocks on average from future
     # testnet value is 800 blocks ~= 1600 minutes (~1.1 days)
     headerChainDiff = 800
+
     # having more internal miner threads helps to reproduce real use cases, like 100 GPU miners
     internalMinersCount = 1
+
     # GPU miners are currently polling at 100ms interval
     internalMinerPollingInterval = 200ms
+
+    # maximum cost of transaction for it to be propagated
+    maxTransactionCost = 800000
```

Comment:
Why each conf file has its own value for this parameter?

Reply:
Block limit is 1M in the testnet, 7M in the mainnet. Thus max tx cost values are different as well.